testsuite/forge/src/backend/local/mod.rs

```diff
@@ -76,7 +78,7 @@ impl LocalFactory {
             })?;
 
         versions.insert(new_version.version.clone(), new_version);
-        Ok(Self::new(versions))
+        Ok(Self::new(versions, ""))
```

Comment:
nit: you could make `swarm_dir` optional if it's not always used?

Reply:
Yeah, it's a little weird, it goes from Option<String> to "" to Option<String> again. And yet I feel like there's a reason why "" makes more sense in the middle data structure.

Reply:
changed to move `Option<String>` around through all the levels and it's better